lib/ansible/modules/cloud/amazon/efs_facts.py

```diff
@@ -136,6 +136,18 @@
     returned: always
     type: str
     sample: "generalPurpose"
+throughput_mode:
```

Comment:
This change should probably be added to efs.py as well

Not sure `version_added` works here does it? Probably needs to be `returned: when botocore is newer than x`.

Reply:
- The relevant descriptions are already present in efs.py
- You're right, ```version_added``` wasn't checked by the tests (2.7 instead of 2.8 didn't trigger the failure) and it doesn't make any sense here - Updated